.circleci/config.yml

```diff
@@ -95,8 +94,6 @@ jobs:
       - run:
           name: run Go linter
           command: |
-            # Identify how many cores it defaults to
```

Comment:
NIT: Is there is any particular reasons that this has been removed? 
Anyway, LGTM to bump the version of the ci-builder :)

Reply:
Flag doesn't exist on whatever version CI builder is using which causes this line to fail